cfme/control/explorer/policies.py

```diff
@@ -421,6 +422,14 @@ def assign_actions_to_event(self, event, actions):
         view.flash.assert_message('Actions for Policy Event "{}" were saved'.format(
             event))
 
+    @property
```

Comment:
@izapolsk  It's checked here https://github.com/ManageIQ/integration_tests/blob/master/cfme/control/explorer/policies.py#L138, but due this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1359909 for now it's broken